api/plugins/action/task_invoke.py

```diff
@@ -0,0 +1,22 @@
+from . import LagoonMutationActionBase, MutationConfig, MutationActionConfig
+from ..module_utils.gql import ProxyLookup
+
+
+class ActionModule(LagoonMutationActionBase):
+
+    actionConfig = MutationActionConfig(
+        name="task_invoke",
+        # Configuration for invoking a registered task.
+        add=MutationConfig(
+            # GraphQL Mutation for invoking a registered task.
+            field="invokeRegisteredTask",
+            # Additional arguments to be allowed when calling the action
+            # plugin.
+            inputFieldArgsAliases=dict(
+                argumentValues=["arguments"],
+                environment=["project"],
```

Comment:
I'm probably not understanding something but can we use `project` as the value for `environment` here?
According to the documentation, GraphQL expects `environment` ID integer, not project ID integer.